### PR TITLE
BZ1634097 -change in corrected files

### PR DESCRIPTION
**Pull request by jhoyt-rh**
BZ is here: https://bugzilla.redhat.com/show_bug.cgi?id=1634097

New PR due to changes in 2 files. Moved added info from one file to the other/correct file.

@anli@redhat.com   PTAL to ensure I understood your update, please.
@gkimetto@redhat.com PTAL

### CODE REVIEW

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -559,11 +559,13 @@ a|This variable configures additional `docker` options within
...
 
-The following example shows the configuration of Docker to use the `json-file` log
-driver, where Docker will rotate between three 1 MB log files:
+To configure the log file, edit the `/etc/sysconfig/docker` file. For example, to set the maximum file size to 1MB and always
+keep the last three log files, append `max-size=1M` and `max-file=3` to the `OPTIONS=` line, ensuring that the values
+maintain the single quotation mark formatting:
 ----
-"--log-driver json-file --log-opt max-size=1M --log-opt max-file=3"
+OPTIONS= --insecure-registry=172.30.0.0/16 --selinux-enabled --log-opt max-size=1M --log-opt max-file=3
 ----
```
Comment (vikram-redhat) [inline install_config/install/advanced_install.adoc]: @anpingli Line 560 says to use json-file and then the example is editing the /etc/sysconfig/docker file instead. Is that correct? Is this different in 3.9 compared to 3.10+?

@jhoyt-rh - the text says "ensuring that the values maintain the single quotation mark formatting", but the actual example doesn't contain any quotations.

Comment (jhoyt-rh) [reply install_config/install/advanced_install.adoc]: THX. re-inserted single quotes, missing in error.

Comment (anpingli) [reply install_config/install/advanced_install.adoc]: I suggest adding --log-driver json-file in OPTIONS, as the max-size and max-file is only available for the log-driver json-file 

OPTIONS='--log-driver json-file  --insecure-registry=172.30.0.0/16 --selinux-enabled --log-opt max-size=1M --log-opt max-file=3'

**Comment by jhoyt-rh** [comment]
Closing this PR due to error made in git. Failed to commit correctly. Will open a new PR to finish this BZ documentation.

**Pull request closed without merging**